src/measure_theory/function/strongly_measurable/basic.lean

```diff
@@ -1425,6 +1425,41 @@ protected lemma indicator [has_zero β]
   ae_strongly_measurable (s.indicator f) μ :=
 (ae_strongly_measurable_indicator_iff hs).mpr hfm.restrict
 
+lemma null_measurable_set_eq_fun {E} [topological_space E] [metrizable_space E]
```

Comment:
Can you give a docstring to both versions of `null_measurable_set_eq_fun` referring to the other one?